crates/nu-command/src/filters/sort_by.rs

```diff
@@ -38,83 +38,17 @@ impl Command for SortBy {
     fn examples(&self) -> Vec<Example> {
         vec![
             Example {
```

Comment:
Just a heads up -- you'll notice when #6796 merges that if we change the examples, we may get an error telling us that the provided examples no longer provide "witnesses" for the declared types. (The rule will be that every type signature variant must have an example, and no example may be inconsistent with all type signature variants.)